docs/contributing/authoring-polyfills.html

```diff
@@ -0,0 +1,181 @@
+{{>header}}
+
+{{>nav this}}
+
+	<div class="o-techdocs-main o-techdocs-content">
+
+		<h1>Authoring polyfills</h1>
+
+		<p>Polyfills are located in the <code>/polyfills</code> directory and organised by feature name.</p>
+
+		<h2 id='naming'>Naming</h2>
+
+		<p>Polyfill directories should be named after their main JavaScript global, and nested subdirectories should be used to descend through objects, e.g. <code>/polyfills/Array/prototype/forEach</code> should be used to hold the polyfill for <code>Array.prototype.forEach</code>.</p>
+
+		<p>Where a polyfill exposes multiple globals, consider splitting it if possible, but if that's not practical, name the polyfill after the most well known of the globals.</p>
+
+		<p>If a feature has no JavaScript API, e.g. support for CSS styling of HTML5 elements, it should be given a descriptive name prefixed with a tilde (~), ie. <code>~html5-elements</code>.</p>
+
+		<h2 id='files'>Files</h2>
+
+		<p>Each polyfill directory (and subdirectory) is structured like this:</p>
+
+		<ul>
+			<li><code>polyfill.js</code>: Required. Code to apply the polyfill behaviour.  See <a href="#configuration">configuration</a>.</li>
+			<li><code>detect.js</code>: A single expression that returns true if the feature is present in the browser (and the polyfill is therefore not required), false otherwise.  Can be an IIFE.  If not present, polyfill cannot be gated (wrapped in a feature-detect).  <strong>Must not have a trailing semicolon</strong>.</li>
```

Comment:
Yes!  Thanks for pointing it out. Looks like Jake jumped in like a champ to fix it.